src/project-css/app/Adapters/ImportAdapter.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Adapters;
+
+use Illuminate\Support\Facades\DB;
+use App\Models\Collection;
+use App\Models\Table;
+use App\Helpers\CSVHelper;
+use App\Helpers\CustomStringHelper;
+
+use Log;
+
+class ImportAdapter {
+    public $savedTkns;
+    public $lastErrRow;
+    public $tkns;
+    public $curArry;
+    public $clmnLst;
+    public $tblNme;
+
+     /**
+     * Takes 2 arrays of tokens and merges them.
+     * Designed around the Rockefeller data their was instances where a
+     * incorrect character caused a break in reading the line. When this 
+     * is detected due to a inconsistent field count we will attempt to 
+     * merge the 2 lines.
+     * @param array $tkns1
+     * @param array $tkns2
+     * @return array
+     */    
+    public function mergeLines() {
```

Comment:
There are no params defined, instead your using cookie tokens.  I'm not seeing the return being set either like your documentation says.